test/spec/ol/worker/version.test.js

```diff
@@ -0,0 +1,32 @@
+import {create} from '../../../../src/ol/worker/version.js';
+import {VERSION} from '../../../../src/ol/util.js';
+
+
+describe('ol/worker/version', function() {
+
+  let worker;
+  beforeEach(function() {
+    worker = create();
+  });
+
+  afterEach(function() {
+    if (worker) {
+      worker.terminate();
+    }
+    worker = null;
+  });
+
+  describe('messaging', function() {
+    it('responds with the version', function(done) {
+      worker.addEventListener('error', done);
```

Comment:
@tschaub, shouldn you fail the test before calling `done`?

Reply:
@gberaudo The `done` callback supports an `Error` as argument. When passed, the test will fail. See https://mochajs.org/#asynchronous-code.